src/operator/src/statement/ddl.rs

```diff
@@ -1224,6 +1228,15 @@ impl StatementExecutor {
     }
 }
 
+/// Fills schema-level options to [CreateTableExpr].
+fn fill_schema_opts(expr: &mut CreateTableExpr, schema: &SchemaNameValue) {
+    if !expr.table_options.contains_key(TTL_KEY)
+        && let Some(ttl) = schema.human_ttl()
+    {
+        expr.table_options.insert(TTL_KEY.to_string(), ttl);
+    }
```

Comment:
I forgot that this will also persist the database ttl in table options.

Is it possible that we add another field `db_options` so we know we won't persist it in table value?